lib/manager/gitlabci/extract.ts

```diff
@@ -7,12 +7,18 @@ import type { ExtractConfig, PackageDependency, PackageFile } from '../types';
 import type { GitlabPipeline } from './types';
 import { replaceReferenceTags } from './utils';
 
+const commentsRe = /^\s*#/;
+const whitespaceRe = /^(?<whitespace>\s*)/;
+const imageRe =
+  /^(?<whitespace>\s*)image:(?:\s+['"]?(?<image>[^\s'"]+)['"]?)?\s*$/;
+const nameRe = /^\s*name:\s+['"]?(?<depName>[^\s'"]+|)['"]?\s*$/;
```

Comment:
```suggestion
const nameRe = /^\s*name:\s+['"]?(?<depName>[^\s'"]+)['"]?\s*$/;
```

What does the `|` match? 🤔

Reply:
Nothing, I guess. I have removed them from the image regex, but it seems I have missed that one